lib/get-content.js

```diff
@@ -0,0 +1,65 @@
+module.exports = async function getContent(page, args) {
+	let blogPage = args.botblogurl;
+	/* CONTENT COLLECTION FROM BLOG POST */
+	await page.goto(blogPage);
+
+	console.log(`Visiting ${blogPage} ...`);
+
+	return await page.evaluate((blogPage) => {
+		getTextFromParagraphs = (paragraphs) => {
+			if (paragraphs) {
+				return [...paragraphs].map((para) => {
+					return para.outerHTML;
+				}).join("");
+			}
+			return "";
+		}
+
+		getParagraphs = (index) => {
+			return document.querySelectorAll(`[data-embertimes-section-fragment="${index}"]`);
```

Comment:
This should allow us to fetch all the text element for a particular section, once https://github.com/ember-learn/ember-blog/pull/149 has landed